src/server/Listener.js

```diff
@@ -151,4 +157,12 @@ function createServer (port, app) {
 	return server;
 }
 
+function createMock (uri) {
+	let keys = []; // eslint-disable-line prefer-const
+	return {
+		keys: keys,
```

Comment:
Btw. the unit tests are a little spotty, so didn't catch this bug:

Start the mock server with the example files:
```
./bin/http-mockserver.js --mocks examples
```

Open http://localhost:3000/listener 
It should show a list of mocks but currently throws:
```
TypeError: Cannot read property 'length' of undefined
    at _.mapValues (/Users/sqren/ts-code/http-mockserver/src/server/Listener.js:142:37)
```